src/Uno.UI.Wasm/ts/Windows/Networking/Connectivity/NetworkInformation.ts

```diff
@@ -0,0 +1,24 @@
+﻿namespace Windows.Networking.Connectivity {
+
+	export class NetworkInformation {
+		private static dispatchStatusChanged: () => number;
+
+		public static startStatusChanged() {
+			window.addEventListener("online", NetworkInformation.networkStatusChanged);
+			window.addEventListener("offline", NetworkInformation.networkStatusChanged);
+		}
+
+		public static stopStatusChanged() {
+			window.removeEventListener("online", NetworkInformation.networkStatusChanged);
+			window.removeEventListener("offline", NetworkInformation.networkStatusChanged);
+		}
+
+		public static networkStatusChanged() {
+			if (window.DeviceMotionEvent) {
+				this.dispatchStatusChanged = (<any>Module).mono_bind_static_method("[Uno] Windows.Networking.Connectivity.NetworkInformation:DispatchStatusChanged");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the parent class name instead of `this` when in a `static` context.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5536782)